MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -103,7 +105,7 @@ - (void)timeUpdateTimer:(NSTimer *)timer {
 
     if (_player.currentItem != nil) {
         AVPlayerItemAccessLog *log = _player.currentItem.accessLog;
-        if (log != nil && log.events.count > 0) {
+        if (_proxy == nil && log != nil && log.events.count > 0) {
```

Comment:
as it is today, `_proxy` is always going to be non-nil. It would be nice to give them a way during instantiation to say whether they want to use the proxy or not, so they can choose one or the other.

Reply:
setSdkOptions API is created and singleton MUXSDKOptions is used for store global value(s)